packages/mobile/src/identity/saga.ts

```diff
@@ -130,10 +116,7 @@ function* fetchKnownAddresses() {
 }
 
 function* watchVerification() {
-  yield takeLeading(Actions.FETCH_VERIFICATION_STATE, handleFetchVerificationState)
-  yield takeLatest(Actions.FEELESS_FETCH_VERIFICATION_STATE, feelessFetchVerificationState)
-  yield takeLatest(Actions.START_VERIFICATION, startVerification)
-  yield takeLatest(Actions.FEELESS_START_VERIFICATION, feelessStartVerification)
+  yield takeLatest(Actions.START_VERIFICATION, startVerificationSaga)
```

Comment:
It will be moved in the second PR together with bunch of other actions: https://github.com/celo-org/wallet/pull/116/files#diff-b05be1e8fafe0db18483ac46fe5b3f14f95d6a02244a0c8b30f21acfe144b20c